lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -468,10 +472,23 @@ describe('AwsInvokeLocal', () => {
       });
     });
 
-    it('throw error when using runtime other than Node.js, Python, Java or Ruby', () => {
-      awsInvokeLocal.options.functionObj.runtime = 'invalid-runtime';
-      expect(() => awsInvokeLocal.invokeLocal()).to.throw(Error);
-      delete awsInvokeLocal.options.functionObj.runtime;
+    it('should call invokeLocalDocker if using runtime provided', () => {
+      awsInvokeLocal.options.functionObj.runtime = 'provided';
+      awsInvokeLocal.options.functionObj.handler = 'handler.foobar';
+      return awsInvokeLocal.invokeLocal().then(() => {
+        expect(invokeLocalDockerStub.calledOnce).to.be.equal(true);
+        expect(invokeLocalDockerStub.calledWithExactly()).to.be.equal(true);
```

Comment:
What's the difference between this and `calledOnce`?

Reply:
`calledOnce` ensures that the function wasn't called more than one time. `calledWithExactly` ensures that it was called _at least once with those exact arguments_, IE, it could be called any number of additional times with other arguments.

Reply:
Makes sense...